coderd/database/dbgen/dbgen.go

```diff
@@ -643,8 +643,8 @@ func ProvisionerJob(t testing.TB, db database.Store, ps pubsub.Pubsub, orig data
 	})
 	require.NoError(t, err, "insert job")
 	if ps != nil {
-		err = provisionerjobs.PostJob(ps, job)
-		require.NoError(t, err, "post job to pubsub")
+		// Advisory, but not essential since acquirer has a background poller to pick up missed jobs.
+		_ = provisionerjobs.PostJob(ps, job)
```

Comment:
This seems overly broad. Most tests will want to know if the notification fails because relying on polling is likely to be flaky.

In your test, could you:

a) not pass the pubsub when you know you have a broken pubsub tests case?
b) only "break" the pubsub after the initial setup that requires `dbgen` is finished?